charts/kyverno/templates/networkpolicy.yaml

```diff
@@ -30,4 +30,43 @@ spec:
   ingress:
     - {}
   {{- end }}
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-apiserver
+spec:
+  podSelector: {}
+  policyTypes:
+  - Ingress
+  - Egress
```

Comment:
Does this need to be wrapped in `{{- if .Values.networkPolicy.openEgress }}` or is it okay to have Egress defined here and no actual egress block below?